src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { CoreStart } from 'opensearch-dashboards/public';
+import {
+  EuiLink,
+  EuiPage,
+  EuiIcon,
+  EuiText,
+  EuiCard,
+  EuiImage,
+  EuiPanel,
+  EuiTitle,
+  EuiSpacer,
+  EuiPageBody,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButtonEmpty,
+  EuiPageContent,
+  EuiSmallButton,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import img from '../../assets/image.png';
+import { WORKSPACE_CREATE_APP_ID } from '../../../common/constants';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+
+export const WorkspaceInitial = () => {
+  const {
+    services: { application, chrome },
+  } = useOpenSearchDashboards<CoreStart>();
+  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
+  const logos = chrome.logos;
+  const createWorkspaceUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, { absolute: true });
+  const settingsAndSetupUrl = application.getUrlForApp('settings_landing', { absolute: true });
+
+  const cards = (
+    <EuiFlexGroup style={{ overflowX: 'scroll', overflowY: 'hidden' }}>
```

Comment:
Nit:
Shall we use `oui-xScroll` in [overflow tools](https://oui.opensearch.org/1.8/#/utilities/css-utility-classes#overflows)? Not sure if it's suitable.

Reply:
Sure, I will use the classname